### ISSUE TITLE
the core net architecture 

### ISSUE THREAD

**Issue by yzhuuu** [zhongwen/predictron#7]
According to appendix A and fig 7 in the original paper, the value net is operated on s^k (the bottom level 0 squares in fig 7), while the gamma net, reward net and lambda net are operated on the first conv layer's output (level 1 squares in fig 7).

In this implementation, the value net is operated on the first conv layer's output, and other 3 nets are operated on the 2nd conv layer's output. (https://github.com/zhongwen/predictron/blob/master/predictron.py#L73) This seems a mismatch with the paper. In short, I believe the value net should not share any conv layers with other branches. 

It might worth to take a second look at those details. Thanks.

**Comment by zhongwen** [comment]
Hi Yan, I did make a mistake here. Will send a fix ASAP. Thanks!

**Comment by zhongwen** [comment]
Hi Yan, please have a look at the [fix_mismatch](https://github.com/zhongwen/predictron/tree/fix_mismatch) branch

**Comment by yzhuuu** [comment]
Thanks. Looks good to me:) 

Feel free to close this thread if the RMSE of the test run looks okay.

**Comment by zhongwen** [comment]
It's running now :) though on less GPUs.

**Comment by zhongwen** [comment]
Seems normal. See the following logs:
```
INFO:multigpu_train:2017-01-19 20:35:59.967757: step 54860, loss = 0.0058, loss_preturns = 0.0026, loss_lambda_preturns = 0.0004 (2062.5 examples/sec; 0.058 sec/batch)
INFO:multigpu_train:2017-01-19 20:36:03.439532: step 54870, loss = 0.0046, loss_preturns = 0.0019, loss_lambda_preturns = 0.0004 (2033.7 examples/sec; 0.059 sec/batch)
INFO:multigpu_train:2017-01-19 20:36:06.920895: step 54880, loss = 0.0097, loss_preturns = 0.0041, loss_lambda_preturns = 0.0007 (2079.7 examples/sec; 0.058 sec/batch)
INFO:multigpu_train:2017-01-19 20:36:14.962271: step 54890, loss = 0.0090, loss_preturns = 0.0041, loss_lambda_preturns = 0.0004 (2046.8 examples/sec; 0.059 sec/batch)
```

I am closing this issue and merge the PR.
